src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -25,6 +25,7 @@ import apiService from 'src/services/api-service';
 import speciesSelectorStorageService from 'src/content/app/species-selector/services/species-selector-storage-service';
 import analyticsTracking from 'src/services/analytics-service';
 import buildAnalyticsObject from 'src/analyticsHelper';
+import { deleteSpeciesAndSave as deleteSpeciesAndSaveBrowser } from 'src/content/app/browser/browserActions';
```

Comment:
`deleteSpeciesAndSaveBrowser` -> `deleteSpeciesInGenomeBrowser`? We will also presumably have `deleteSpeciesInEntityViewer` etc. later on.

Reply:
That's done 👍